virt/release_notes/virt-4-17-release-notes.adoc

```diff
@@ -152,7 +152,8 @@ Deprecated features are included in the current release and supported. However,
 
 Removed features are those that were deprecated in earlier releases. They are now removed from {VirtProductName} and are no longer supported.
 
-No features were removed in this release.
+//CNV-45783
+* CentOS 7 and CentOS Stream 8 are now in the End of Life phase. As a consequence, the container ímages for these operating systems have been removed from OpenShift Virtualization and are no longer link:https://access.redhat.com/articles/4234591#community[community supported].
```

Comment:
😅 Sorry I only caught this when looking at another version, but the i here is accented for some reason:
```suggestion
* CentOS 7 and CentOS Stream 8 are now in the End of Life phase. As a consequence, the container images for these operating systems have been removed from OpenShift Virtualization and are no longer link:https://access.redhat.com/articles/4234591#community[community supported].
```